darwin/PlatformHelpers.c

```diff
@@ -59,7 +59,11 @@ bool Platform_KernelVersionIsBetween(KernelVersion lowerBound, KernelVersion upp
 }
 
 void Platform_getCPUBrandString(char* cpuBrandString, size_t cpuBrandStringSize) {
+#ifdef __POWERPC__
+   if (sysctlbyname("hw.cpusubtype", cpuBrandString, &cpuBrandStringSize, NULL, 0) == -1) {
+#else
    if (sysctlbyname("machdep.cpu.brand_string", cpuBrandString, &cpuBrandStringSize, NULL, 0) == -1) {
+#endif
```

Comment:
I think this one might be a bit more obvious in its intention?
```suggestion
   if (sysctlbyname("machdep.cpu.brand_string", cpuBrandString, &cpuBrandStringSize, NULL, 0) == -1) {
#ifdef __POWERPC__
      if (sysctlbyname("hw.cpusubtype", cpuBrandString, &cpuBrandStringSize, NULL, 0) != -1)
         return;
#endif
```